change/@itwin-tree-widget-react-2b487f08-fb59-4b3b-80cf-514f79487c71.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Models Tree: reduce scope of models that are queried as available.",
```

Comment:
Adjusted your recommendation a bit to ""Models Tree: For show/hide all actions.." -> "Models Tree: For all header actions..." because the modified query results are passed to all rendered buttons, not just show/hide all